e2e/support/helpers/e2e-ui-elements-helpers.js

```diff
@@ -70,6 +70,10 @@ export function filterWidget() {
   return cy.get("fieldset");
 }
 
+export function clearFilterWidget(index = 0) {
+  return filterWidget().eq(index).find(".Icon-close").click();
```

Comment:
We use `.icon` to find our icons.

```suggestion
  return filterWidget().eq(index).icon("close").click();
```